collect_app/src/main/java/org/odk/collect/android/tasks/InstanceGoogleSheetsUploader.java

```diff
@@ -273,6 +283,27 @@ private void insertRow(TreeElement element, String parentKey, String key, File i
         }
     }
 
+    /**
+     * Adds titles ending with "-altitude" or "-accuracy" that may have been added to the
```

Comment:
This is for the case where a user manually adds the altitude and accuracy columns, right? It took me a while to get that and I think maybe just adding the word "manually" to "may have been manually added to the..." would help with that.

Reply:
That makes more sense, I'll change to "Adds titles ending with "-altitude" or "-accuracy" that may have been manually added to the ..."